src/Events.sol

```diff
@@ -12,6 +12,10 @@ contract Events {
     event CollateralStatus(address indexed account, address indexed collateral, bool enabled);
     event ControllerStatus(address indexed account, address indexed controller, bool enabled);
     event CallWithContext(
-        address indexed caller, address indexed targetContract, address indexed onBehalfOfAccount, bytes4 selector
+        address indexed caller,
+        uint152 indexed onBehalfOfAddressPrefix,
```

Comment:
It would look nicer on etherscan if address prefixes are exposed as `bytes19` instead of `uint152` because you'd see the first 19 bytes matching, instead of a huge decimal number. Too bad we didn't think of this earlier, because if we change it here we'd probably also want to change it in the rest of the interfaces. Probably not worth it.

Reply:
actually, that's a very good idea! I didn't notice that. it wasn't much work, look at the commit below